omegaconf/omegaconf.py

```diff
@@ -226,7 +226,10 @@ def _create_impl(  # noqa F811
                     ref_type = OmegaConf.get_type(obj)
```

Comment:
what is this actually doing?
what will ref_type be if is_primitive_list(obj)?
what will it be if OmegaConf.is_list(obj)?

Reply:
My bad, I changed it because I was trying another thing.

Reply:
It's good to review your own code critically, making sure you can justify each change.